crates/red_knot_module_resolver/src/resolver.rs

```diff
@@ -442,6 +442,49 @@ pub(crate) mod internal {
     }
 }
 
+/// Modules that are builtin to the Python interpreter itself.
+///
+/// When these module names are imported, standard module resolution is bypassed:
+/// the module name always resolves to the stdlib module,
+/// even if there's a module of the same name in the workspace root
+/// (which would normally result in the stdlib module being overridden).
+///
+/// TODO(Alex): write a script to generate this list,
+/// similar to what we do in `crates/ruff_python_stdlib/src/sys.rs`
+const BUILTIN_MODULES: &[&str] = &[
```

Comment:
rather than being a `const &[&str]`, this could also possibly be a `Lazy<HashSet<& 'static str>>`, which might make containment checks faster. But we can probably optimize this in the followup where we switch to generating this list?

Reply:
We should use `static` here. `const` inlines the variable on every use. we don't want that.

Reply:
done!